set.mm

```diff
@@ -425567,8 +425699,16 @@ that the number n of vertices in G is exactly k(k-1)+1.".  Variant of
     $( Value of operation ` F ` , mapping a vertex ` v ` and a positive integer
        ` n ` to the "(For a fixed vertex v, let f(n) be the number of) walks
        from v to v of length n" according to definition 5 in [Huneke] p. 2.
-       (Contributed by Alexander van der Vekens, 14-Sep-2018.)  (Revised by AV,
-       28-May-2021.) $)
+
```

Comment:
Why not simply write:
> Value of operation ` F ` , mapping a vertex ` v ` and a positive integer ` n ` to the number of walks from `v` to `v` of length `n`" according to definition 5 in [Huneke] p. 2.

(sorry this is not even in your change set!)

Reply:
It seems this defines the _set_ of walks, not the number of walks.
Also, it seems this defines the set of _closed_ walks, not just walks.